js/releases/index.js

```diff
@@ -18,7 +18,16 @@ const SOURCEMAPS_SCHEMA = require('./options/uploadSourcemaps');
  * Manages releases and release artifacts on Sentry.
  * @namespace SentryReleases
  */
-module.exports = {
+class Releases {
+  /**
+   * Creates a new `Releases` instance.
+   *
+   * @param {Object} [options] More options to pass to the CLI
+   */
+  constructor(options) {
+    this.options = options;
```

Comment:
Can we default to `{}` in params here just in case? It will be a very soft guard for `this.options.silent` accessing undefined.